### PR TITLE
chore: add can-i-merge badge docs

### PR DESCRIPTION
**Pull request by mefellows**
Needs a new release of the Pact Broker gem before we release this, and an update to the version in this docs page.

### CODE REVIEW

**Comment by mefellows** [comment]
Yousaf - FYI. New `can-i-merge` badge feature.

I updated the docs for consistency with the other docs on the page, but the `test` tenant needs some work/cleanup. I think we tackle this separately however.

**Comment by pahnin** [comment]
LGTM

**Comment by YOU54F** [comment]
Agreed at tackling sep, looks great, will get the pb gem released today and doing a release of pb client anyway.

hmm at the build failure

File: website/docs/pact_broker/advanced_topics/provider_verification_badges.md [hunk 1]
```diff
@@ -32,13 +32,31 @@ Requires version 2.62+ of the Pact Broker.
...
 ```
 
-[![Can I deploy Foo status](https://test.pactflow.io/pacticipants/FrontendWebsite/latest-version/test/can-i-deploy/to/test/badge?label=my+custom+label+here)](https://test.pactflow.io/pacticipants/FrontendWebsite/latest-version/test/can-i-deploy/to/test)
+[![Can I deploy Foo status](https://test.pactflow.io/pacticipants/pactflow-example-consumer/latest-version/prod/can-i-deploy/to/production/badge?label=my+custom+label+here)](https://test.pactflow.io/pacticipants/pactflow-example-consumer/latest-version/prod/can-i-deploy/to/production)
+
+To set a custom label for the badge, set the `label` query parameter. eg `?label=my+custom+label+here`.
+
+## Can I Merge badge
+
+[![Can I Merge Status](/img/can-i-merge-badge.svg)](https://test.pactflow.io)
```
Comment (YOU54F) [inline website/docs/pact_broker/advanced_topics/provider_verification_badges.md:47]: add image to codebase

File: website/docs/pact_broker/advanced_topics/provider_verification_badges.md [hunk 2]
```diff
@@ -6,19 +6,19 @@ _See the the_ [_Badges_](/pact_broker/configuration/features#badges) _section of
...
+[![Pact Status](https://test.pactflow.io/pacticipants/pactflow-example-consumer/branches/master/latest-version/can-i-deploy/to-environment/prod/badge)](https://test.pactflow.io/pacticipants/pactflow-example-consumer/branches/master/latest-version/can-i-deploy/to-environment/production)
 
 ## Can I Deploy badge
 
 [![Can I Deploy Status](/img/can-i-deploy-badge.svg)](https://test.pactflow.io)
 
 ### Using branches and environments
 
-Requires version 2.94+ of the Pact Broker
+Requires version 2.XYZ of the Pact Broker
```
Comment (YOU54F) [inline website/docs/pact_broker/advanced_topics/provider_verification_badges.md:17]: [2.111.0](https://github.com/pact-foundation/pact_broker/releases/tag/v2.111.0) 

I think we should retain this as 2.94+ for can-i-deploy badge, and add 2.111.0 under the can-i-merge badge, to be clear.

File: website/docs/pact_broker/advanced_topics/provider_verification_badges.md [hunk 3]
```diff
@@ -32,13 +32,31 @@ Requires version 2.62+ of the Pact Broker.
...
-[![Can I deploy Foo status](https://test.pactflow.io/pacticipants/FrontendWebsite/latest-version/test/can-i-deploy/to/test/badge?label=my+custom+label+here)](https://test.pactflow.io/pacticipants/FrontendWebsite/latest-version/test/can-i-deploy/to/test)
+[![Can I deploy Foo status](https://test.pactflow.io/pacticipants/pactflow-example-consumer/latest-version/prod/can-i-deploy/to/production/badge?label=my+custom+label+here)](https://test.pactflow.io/pacticipants/pactflow-example-consumer/latest-version/prod/can-i-deploy/to/production)
+
+To set a custom label for the badge, set the `label` query parameter. eg `?label=my+custom+label+here`.
+
+## Can I Merge badge
+
+[![Can I Merge Status](/img/can-i-merge-badge.svg)](https://test.pactflow.io)
+
+### Using branches and environments
```
Comment (YOU54F) [inline website/docs/pact_broker/advanced_topics/provider_verification_badges.md:49]: Using branches.

environments not relevant to this section

maybe point to configuring main branch ref docs? https://docs.pact.io/pact_broker/branches#pacticipant-main-branch-property

File: website/docs/pact_broker/advanced_topics/provider_verification_badges.md [hunk 4]
```diff
@@ -6,19 +6,19 @@ _See the the_ [_Badges_](/pact_broker/configuration/features#badges) _section of
 
 If you are publishing [Provider verification results](provider_verification_results.md) to your Pact Broker \(v2.3.0+\), you can also display the verification status in your READMEs using a shiny badge like this one:
 
-[![Pact Status](https://test.pactflow.io/pacticipants/FrontendWebsite/branches/test/latest-version/can-i-deploy/to-environment/test/badge)](https://test.pactflow.io/pacticipants/FrontendWebsite/branches/test/latest-version/can-i-deploy/to-environment/test)
+[![Pact Status](https://test.pactflow.io/pacticipants/pactflow-example-consumer/branches/master/latest-version/can-i-deploy/to-environment/prod/badge)](https://test.pactflow.io/pacticipants/pactflow-example-consumer/branches/master/latest-version/can-i-deploy/to-environment/production)
```
Comment (YOU54F) [inline website/docs/pact_broker/advanced_topics/provider_verification_badges.md:9]: For branches (can i merge badge), For environments (can-i-deploy badge)

File: website/docs/pact_broker/advanced_topics/provider_verification_badges.md [hunk 5]
```diff
@@ -6,19 +6,19 @@ _See the the_ [_Badges_](/pact_broker/configuration/features#badges) _section of
 
 If you are publishing [Provider verification results](provider_verification_results.md) to your Pact Broker \(v2.3.0+\), you can also display the verification status in your READMEs using a shiny badge like this one:
```
Comment (YOU54F) [inline website/docs/pact_broker/advanced_topics/provider_verification_badges.md:7]: like these (if we decide to show both badge styles in the top most section?)

File: website/docs/pact_broker/advanced_topics/provider_verification_badges.md (hunk 2 above)
Comment (mefellows) [reply website/docs/pact_broker/advanced_topics/provider_verification_badges.md:17]: yeah, this was a copy/pasta error. Resolved.

File: website/docs/pact_broker/advanced_topics/provider_verification_badges.md (hunk 4 above)
Comment (mefellows) [reply website/docs/pact_broker/advanced_topics/provider_verification_badges.md:9]: Done - thanks!

**Pull request merged**